protex/system.py

```diff
@@ -11,7 +11,7 @@ class IonicLiquidTemplates:
     def __init__(self, states: list, allowed_updates: dict[frozenset]) -> None:
```

Comment:
tests are failing here with:
```    
def __init__(self, states: list, allowed_updates: dict[frozenset]) -> None:
E   TypeError: 'type' object is not subscriptable 
```
I think the `frozenset` has to be removed here, or type hint with `Dict[frozenset]`, following this suggestion:
https://stackoverflow.com/questions/52593803/how-to-set-the-python-type-hinting-for-a-dictionary-variable

Reply:
Yes, thanks for the change